### ISSUE TITLE
[A11Y][Text input] Trailing action button should have its own accessibility label

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1087]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14

### OS Version

iOS26

### App version

0.21

### App build number

.

### App build type

beta

### Describe the issue

Trailing action button is grouped with the text input, several problems with that:
 - the accessibility name of the button is very long and not consistent with the action (input label + content + hint)
 - possible issue with full keyboard access or other assistive techs to reach the button
 - the user can't find the button by exploring the screen (keep finger down and move)
 

<img width="418" height="91" alt="" src="https://github.com/user-attachments/assets/c52bf6f2-1c09-419b-87f9-07819b78e073" />

### Expected Behavior

- put the button out of the input group
- set a specific and customisable `.accessibilityLabel` to the trailing action button

### Accessibility settings

Voice Over and FKA

**Comment by ludovic35** [comment]
L'idée est de faire d'in point de vue  VoiceOver que le trailing action est un bouton à coté du text input

**Comment by ludovic35** [comment]
set empty label in the accessibilityValue

`let emptyValueDescription = text.wrappedValue.isEmpty ? "core_textInput_empty_a11y".localized() : ""`

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765490917**
- Build tag: **3575bbd**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765577305**
- Build tag: **3492061**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765836505**
- Build tag: **9da812c**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765922908**
- Build tag: **8ba3d47**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
